### PR TITLE
Update phpstan

### PR DESCRIPTION
**Pull request by Prokyonn**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | fixes #issuenum
| Related issues/PRs | #issuenum
| License | MIT

#### What's in this PR?

Update phpstan.

### CODE REVIEW

File: Admin/AutomationAdmin.php [hunk 1]
```diff
@@ -76,7 +76,7 @@ public function configureViews(ViewCollection $viewCollection): void
     }
 
     /**
-     * {@inheritdoc}
+     * @return mixed[]
```
Comment (alexander-schranz) [inline Admin/AutomationAdmin.php:79]: We should contribute this back to `sulu/sulu` so we don't have this issue in every bundle.

If we are correct we would need to go with something like:

```
@return array<string, array<string, array<string, array<int, string>>>>
```

but think that would be to complex 🙈

File: Controller/TaskController.php [hunk 2]
```diff
@@ -379,7 +383,7 @@ private function getFieldDescriptors(string $type = null): array
     }
 
     /**
-     * {@inheritdoc}
+     * @return string
```
Comment (alexander-schranz) [inline Controller/TaskController.php:386]: same for this we should contribute this back to sulu release/2.0 branch (SecuredControllerInterface)

File: Entity/Task.php [hunk 3]
```diff
@@ -163,15 +163,15 @@ public function setEntityId(string $entityId): self
         return $this;
     }
 
-    public function getTaskId(): string
+    public function getTaskId(): ?string
```
Comment (alexander-schranz) [inline Entity/Task.php:166]: @wachterjohannes can you confirm this change?

**Review by alexander-schranz — APPROVED** [review]

File: Entity/Task.php (hunk 3 above)
Comment (alexander-schranz) [reply Entity/Task.php:166]: Okay seem like onDelete can make this null: https://github.com/sulu/SuluAutomationBundle/blob/2cfc3a7122a96939d2f203f6fe54bcb74e2cfd46/Resources/config/doctrine/Task.orm.xml#L25

Comment (wachterjohannes) [reply Entity/Task.php:166]: yes - it should be nullable - if the ta_task table was cleared (in favor of memory)

**Pull request merged**